testnet/tools/nns-tools/README.md

```diff
@@ -408,33 +408,46 @@ At a high level, there are two sub-step here:
 Generate a mostly pre-populated proposal text file:
 
 ```bash
+# Fill these in.
```

Comment:
I wonder if you couldn't just make an actual script for all this?

Reply:
True but scripts in docs tend to get out of date more than scripts that are scripts, b/c people don't use the doc directly in general.

Reply:
NCR, I think it would be great to have a script that actually does this, so we don't have to hand build one every time, and you already did 80%+ of the work :)

Reply:
Yes, ofc, that is ideal. I'm trying to time box release work. I can look at this again next time the rotation visits me again.